incubator/hnc/docs/user-guide/concepts.md

```diff
@@ -530,6 +529,25 @@ namespace if this annotation already exists. The two are mutually exclusive.
 We are considering replacing this with the standard
 `app.kubernetes.io/managed-by` label in the future.
 
+<a name="excluded-namespace-label">
+
+#### hnc.x-k8s.io/excluded-namespace (label on namespaces)
+
+This label should be added to namespaces such as `kube-system` and `kube-public`
+so that HNC's validating webhook cannot accidentally prevent operations in these
+namespaces and block critical cluster operations.
+
+Before installing HNC, users can customize the excluded namespace list in the HNC
+deployment with a container arg called `excluded-namespace` in
+`config/manager/manager.yaml` and then set this label on the excluded namespaces.
+Setting this label on namespaces that are not
+listed in the HNC deployment as an `excluded-namespace` is not allowed.
+
+As of March 2021, the default excluded namespaces listed in [config/manager/manager.yaml](https://github.com/kubernetes-sigs/multi-tenancy/blob/master/incubator/hnc/config/manager/manager.yaml)
```

Comment:
I kind of feel this should be called out in the installation section or linked from their. As it's a pre/post installation action that is somewhat hidden away but also highly recommended for users to do.

Reply:
Yeah, it should probably be mentioned [here](https://github.com/kubernetes-sigs/multi-tenancy/blob/master/incubator/hnc/docs/user-guide/how-to.md#install-or-upgrade-hnc-on-a-cluster), but it will be more important to call it out in the release notes, which is where the latest installation instructions are.

Reply:
Filed https://github.com/kubernetes-sigs/multi-tenancy/issues/1466 to add excluded namespace configuration to v0.8 release notes and the installation instruction.